dist/images/ovndb-raft-functions.sh

```diff
@@ -272,12 +343,14 @@ ovsdb-raft() {
   echo "=============== ${db}-ovsdb-raft ========== RUNNING"
 
   if [[ "${POD_NAME}" == "ovnkube-db-0" ]]; then
-    # set the election timer value before other servers join the cluster and it can
-    # only be set on the leader so we must do this in ovnkube-db-0 when it is still
-    # a single-node cluster
-    set_election_timer ${db} ${election_timer}
-    if [[ ${db} == "nb" ]]; then
-      set_northd_probe_interval
+    if ${initial_raft_create}; then
+      # set the election timer value before other servers join the cluster and it can
+      # only be set on the leader so we must do this in ovnkube-db-0 when it is still
+      # a single-node cluster
+      set_election_timer ${db} ${election_timer}
+      if [[ ${db} == "nb" ]]; then
+        set_northd_probe_interval
+      fi
     fi
     # set the connection and disable inactivity probe, this deletes the old connection if any
     set_connection ${db} ${port}
```

Comment:
Yeah, I saw the comment, but I didn't think it would have any side-effect if we called set_connection every time. The third bullet about subsequent restarts during lifetime of the pod is explained as
```
# In the third case,                                                                                          
# the pod is a part of mutli-pods cluster and may not be a leader and the connection information should                                                                                       
# have already been set, so we don't care.
```

Reply:
OK I will make the change.